### PR TITLE
add project member handler

### PR DESCRIPTION
**Pull request by runzexia**

### CODE REVIEW

**Comment by runzexia** [comment]
ping @MagicSong

File: pkg/service/projects/project_member_handler.go [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+	projectId := r.PathParams["id"]
+	operator := userutils.GetUserNameFromRequest(r)
+	request := &AddProjectMemberRequest{}
+	err := r.DecodeJsonPayload(request)
+	if err != nil {
+		logger.Error("%v", err)
+		rest.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	if govalidator.IsNull(request.Username) {
```
Comment (MagicSong) [inline pkg/service/projects/project_member_handler.go]: i think it is better to return HttpStatusBadRequest here

Comment (MagicSong) [reply pkg/service/projects/project_member_handler.go]: The same below

**Comment by MagicSong** [comment]
@runzexia  [merge]

**Pull request merged**